### PR TITLE
Implemented the following module skeletons

### PR DESCRIPTION
**Pull request by mohaideenisync**
Implemented the following module skeletons

1. Dashboard
2. Line Monitoring
3. Device Management
4. Reports

### CODE REVIEW

File: src/main/webapp/resources/app/app.component.html [hunk 1]
```diff
@@ -1,7 +1,5 @@
-
-    
 <app-header *ngIf="!isTreeCollapsed"></app-header>
-
+<app-sub-header></app-sub-header>
```
Comment (siva-sncr) [inline src/main/webapp/resources/app/app.component.html]: this should have the condition check for user session

File: src/main/webapp/resources/app/modules/devices/devices.component.ts [hunk 2]
```diff
@@ -2,15 +2,18 @@ import { Component, OnInit } from '@angular/core';
 import { AngularMultiSelectModule } from 'angular2-multiselect-dropdown/angular2-multiselect-dropdown';
 import { Datas } from './data';
 import {DevicesService} from './services/devices.service';
+import {UtilityService} from '../..//services/utility.service';
```
Comment (siva-sncr) [inline src/main/webapp/resources/app/modules/devices/devices.component.ts]: extra backslash before "services" string

**Review by siva-sncr — COMMENTED** [review]
Make sure we follow proper format for file names, example:

1) line.monitoring.component.ts should be line-monitoring.component.ts
2) inactivedevices.component.ts should be inactive-devices.component.ts

**Comment by mohaideenisync** [comment]
@siva-sncr,  Covered review point changes.

**Pull request merged**